### DISCUSSION TITLE
Issue? Loading Chunk 160 failed

### DISCUSSION THREAD

**Discussion by Fringie** [microsoft-search/pnp-modern-search#3119] [General]
Helloo,

We've got a bit of a strange issue, I'm hoping some of you may have encountered the same or a similar issue.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/7203124/a1dfa03a-7ae7-4526-9f33-d0fddc5a6e2a)

And we can see the file has failed to load here..
![MicrosoftTeams-image (2)](https://github.com/microsoft-search/pnp-modern-search/assets/7203124/e8cd6066-eb15-45f0-b038-d6d9ecd09dab)

Reloading the cache/page 'resolves' the issue. The confusing part is, it's affecting some users but not others. The only time I've seen these types of issues if with conditional access blocking the CDN but I don't think is the case because it's only applying to a handful of users.

Any thoughts or suggestions?
**Edit** This issue happens when you navigate to the page via the navbar, but if you navigate to the site via a link the issue does not occur. Maybe this issue does occur to all users.

**Comment by wobba** [comment]
If it's affecting some it sounds like a caching issue. Did you recently upgrade the web parts?

**Reply by Fringie** [reply]
Thanks for your input.

No, we haven't upgraded the webparts, they were setup with V4.1 quite a while ago and this issue has only recently started happening.

If it's cache related, it's worth mentioning we have three different versions of PNP Modern search installed. A few weeks ago I set up pnp modern search on our intranet homepage using V4.8, these issues are occurring on a different site which uses V4.1.

**Reply by wobba** [reply]
If nothing has changed in your deployment I would lean to it being an SPFx issue as internals can have changed. Not a web part issue per say.
